pyproject.toml

```diff
@@ -0,0 +1,78 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "lsst-meas-extensions-multiprofit"
+authors = [
+    {name="Rubin Observatory Data Management", email="dm-admin@lists.lsst.org"},
+]
+description = "Tasks and pipelines for running multiprofit with the LSST Butler"
+readme = "README.rst"
+requires-python = ">=3.10"
+keywords = ["astronomy", "lsst", "galaxy"]
+license = {file = "LICENSE"}
+classifiers = [
+    "Programming Language :: Python :: 3",
+]
+dependencies = [
+    "astropy",
+    "galsim",
+    "gauss2d",
+    "gauss2dfit",
+    "lsst-multiprofit",
+    "matplotlib",
+    "numpy",
+    "pydantic",
+    "pytest",
```

Comment:
pytest is presumably not a run time dependency? You can add a separate optional dependencies entry in the the file and declare test dependencies in there.

```toml
[project.optional-dependencies]
test = [
    "pytest >= 3.2",
]
```

Reply:
Is that v3.2 minimum from a feature we're using in the stack or a compatibility issue?

More generally I couldn't figure out how to set minimum versions of anything for standalone packages like multiprofit, beyond that I made a py3.8 conda env and confirmed that it wouldn't work. Is there anything else I could/should do other than follow changes to other stack packages like resources/daf_butler?

Reply:
The 3.2 was just an example.  It's likely wildly out of date. The point is to separate runtime deps from testing deps. Minimum versions are done the same way `pytest <5,>=3.2` or something.